### PR TITLE
[New] Update phaser auto update config and homepage

### LINKED ISSUE #12874: [Request] Update phaser
phaser already v3.10.1: https://github.com/photonstorm/phaser

but still only v2.6.2:
https://cdnjs.com/libraries/phaser

### PR DESCRIPTION
**Pull request by sashberd**
The build folder was changed to dist
close #12874

Pull request for issue: #12874
Related issue(s): # #

# Git commit checklist
 * [X] The first line of commit message is less then 50 chars; clean, clear and easy to understand.
 * [x] The parent of the commit(s) in the PR is not older than 3 days.
 * [x] Pull request is sent from a non-master branch with a meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/phaser/package.json [hunk 1]
```diff
@@ -25,7 +25,7 @@
   "npmName": "phaser",
   "npmFileMap": [
     {
-      "basePath": "build",
+      "basePath": "dist",
       "files": [
         "**/!(*.php)"
```
Comment (extend1994) [inline ajax/libs/phaser/package.json]: Let's use `**/*`

Comment (extend1994) [reply ajax/libs/phaser/package.json]: Forget to mention, please also help update homepage to https, thanks!

**Comment by sashberd** [comment]
@extend1994 The PR was updated. FYR

File: ajax/libs/phaser/package.json [hunk 2]
```diff
@@ -21,13 +21,13 @@
     "url": "https://github.com/photonstorm/phaser.git"
   },
   "license": "MIT",
-  "author": "Richard Davey <rdavey@gmail.com> (http://www.photonstorm.com)",
+  "author": "Richard Davey <rdavey@gmail.com> (https://www.photonstorm.com)",
```
Comment (extend1994) [inline ajax/libs/phaser/package.json]: author's page doesn't support https.

File: ajax/libs/phaser/package.json [hunk 3]
```diff
@@ -3,7 +3,7 @@
   "filename": "phaser.min.js",
   "version": "2.6.2",
   "description": "A fast, free and fun HTML5 Game Framework for Desktop and Mobile web browsers.",
-  "homepage": "http://phaser.io/",
+  "homepage": "https://phaser.io/",
```
Comment (extend1994) [inline ajax/libs/phaser/package.json]: Please remember to separate auto-update update and other parts, thanks.

File: ajax/libs/phaser/package.json (hunk 2 above)
Comment (sashberd) [reply ajax/libs/phaser/package.json]: Hmmm. Strange.... From my side it is working...
screen shot 2018-07-18 at 8 53 08

Comment (extend1994) [reply ajax/libs/phaser/package.json]: @sashberd Your screenshot is the homepage?

**Comment by sashberd** [comment]
@extend1994 The PR was updated. FYR

**Comment by extend1994** [comment]
@sashberd Use `close` in `Update auto update config`  commit mesage please.

**Comment by sashberd** [comment]
@extend1994 Done

**Pull request merged**